src/utils/compat.ts

```diff
@@ -26,7 +26,10 @@ import {
 import { Network as LiquidNetwork } from "liquidjs-lib/src/networks";
 
 import { network } from "../config";
-import { LBTC } from "../consts";
+import { BTC, LBTC } from "../consts";
+
+const btcPrefixes = ["bc", "tb", "bcrt"];
+const liquidPrefixes = ["lq", "ex", "tlq", "tex", "el", "ert"];
```

Comment:
Don't hardcode those; if anything, get them from the configure network. Also those are only the b(l)ech32(m) ones?